AlphaWallet/KeyManagement/EtherKeystore.swift

```diff
@@ -53,56 +53,56 @@ open class EtherKeystore: Keystore {
     private let defaultKeychainAccessUserPresenceNotRequired: KeychainSwiftAccessOptions = .accessibleWhenUnlockedThisDeviceOnly(userPresenceRequired: false)
     private let userDefaults: UserDefaults
     private var watchAddresses: [String] {
-        set {
-            let data = NSKeyedArchiver.archivedData(withRootObject: newValue)
-            return userDefaults.set(data, forKey: Keys.watchAddresses)
-        }
         get {
             guard let data = userDefaults.data(forKey: Keys.watchAddresses) else {
                 return []
             }
             return NSKeyedUnarchiver.unarchiveObject(with: data) as? [String] ?? []
         }
-    }
-
-    private var ethereumAddressesWithPrivateKeys: [String] {
         set {
             let data = NSKeyedArchiver.archivedData(withRootObject: newValue)
-            return userDefaults.set(data, forKey: Keys.ethereumAddressesWithPrivateKeys)
+            return userDefaults.set(data, forKey: Keys.watchAddresses)
         }
+    }
+
+    private var ethereumAddressesWithPrivateKeys: [String] {
         get {
             guard let data = userDefaults.data(forKey: Keys.ethereumAddressesWithPrivateKeys) else {
                 return []
             }
             return NSKeyedUnarchiver.unarchiveObject(with: data) as? [String] ?? []
         }
-    }
-
-    private var ethereumAddressesWithSeed: [String] {
         set {
             let data = NSKeyedArchiver.archivedData(withRootObject: newValue)
-            return userDefaults.set(data, forKey: Keys.ethereumAddressesWithSeed)
+            return userDefaults.set(data, forKey: Keys.ethereumAddressesWithPrivateKeys)
```

Comment:
@James-Sangalli wow. The diff makes it look so scary. It's actually just swapping the `get { }` and `set {}` blocks..

Reply:
Hang on. Let me check just to be sure. This looks to scary. Maybe I copied and paste wrongly.

Reply:
Ok. Looks right to me. The diff is incredibly misleading

Reply:
can improve this flow?

Reply:
What do you mean by "flow"?

Reply:
I meant the I was just swapping the positions of the `get { }` and `set { }` blocks because SwiftLint prefers the `get { }` to appear before `set {} `. But if you look at the diff, it looks like the logic of the code has changed:

```
Keys.ethereumAddressesWithSeed
->
Keys.ethereumAddressesWithPrivateKeys
```